integrationTests/vm/txsFee/multiShard/scCalls_test.go

```diff
@@ -129,6 +151,15 @@ func TestScCallExecuteOnSourceAndDstShardInvalidOnDst(t *testing.T) {
 	developerFees := testContextSource.TxFeeHandler.GetDeveloperFees()
 	require.Equal(t, big.NewInt(0), developerFees)
 
+	intermediateTxs := testContextSource.GetIntermediateTransactions(t)
```

Comment:
typo "increment" on line 135?

Reply:
is not typo because I want to call a method that does not exist.